app/tests/integration/nimbus/conftest.py

```diff
@@ -143,10 +143,45 @@ def experiment_name(request):
     return f"{request.node.name[:75]}-{str(uuid.uuid4())[:4]}"
 
 
+@pytest.fixture(name="load_experiment_outcomes")
+def fixture_load_experiment_outcomes():
+    """Fixture to create a list of outcomes based on the current configs."""
+    outcomes = {"firefox_desktop": "", "fenix": "", "firefox_ios": ""}
+    base_path = "/code/app/experimenter/outcomes/jetstream-config-main/outcomes"
+
+    for k in list(outcomes):
+        outcomes[k] = [
+            name.split("_")[0].rsplit(".")[0]
+            for name in os.listdir(f"{base_path}/{k}")
+            if "example" not in name
+        ]
+    return outcomes
+
+
 @pytest.fixture
-def default_data(application, experiment_name):
+def default_data(application, experiment_name, load_experiment_outcomes):
     feature_config_id = APPLICATION_FEATURE_IDS[application]
 
+    if "firefox_desktop" in str(application).lower():
+        metrics_data = BaseExperimentMetricsDataClass(
+            primary_outcomes=[load_experiment_outcomes["firefox_desktop"][0]],
+            secondary_outcomes=[load_experiment_outcomes["firefox_desktop"][1]],
+        )
+    elif "fenix" in str(application).lower():
+        metrics_data = BaseExperimentMetricsDataClass(
+            primary_outcomes=[load_experiment_outcomes["fenix"][0]],
+            secondary_outcomes=[load_experiment_outcomes["fenix"][1]],
+        )
+    elif "ios" in str(application).lower():
+        metrics_data = BaseExperimentMetricsDataClass(
+            primary_outcomes=[load_experiment_outcomes["firefox_ios"][0]],
+            secondary_outcomes=[load_experiment_outcomes["firefox_ios"][1]],
+        )
+    else:
+        metrics_data = BaseExperimentMetricsDataClass(
+            primary_outcomes=[], secondary_outcomes=[]
+        )
```

Comment:
This looks like it could be better represented with a dict?

Reply:
Hmm, I think I can see what you mean. I would still have to do a application check but it would be cleaner. Good idea.